package.json

```diff
@@ -54,5 +55,8 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "prettier": "2.8.1",
     "tailwindcss": "^3.2.4"
+  },
+  "resolutions": {
```

Comment:
There are no values in this element. Can it be removed?

Reply:
Done